arch/arm/core/cortex_m/Kconfig

```diff
@@ -35,20 +35,60 @@ config ISA_THUMB2
 	  technology is featured in the  processor, and in all ARMv7
 	  architecture-based processors.
 
+config CPU_CORTEX_M_HAS_PRIMASK
```

Comment:
What's the point of this config option, not seeing it used, and if it exists for all v6, v7, v8 what value does it provide?